experimental/packages/opentelemetry-sdk-metrics-base/test/export/TestMetricExporter.ts

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { ExportResult, ExportResultCode } from '@opentelemetry/core';
+import { AggregationTemporality, PushMetricExporter, ResourceMetrics } from '../../src';
+
+export class TestMetricExporter implements PushMetricExporter {
+  resourceMetricsList: ResourceMetrics[] = [];
+  export(resourceMetrics: ResourceMetrics, resultCallback: (result: ExportResult) => void): void {
+    this.resourceMetricsList.push(resourceMetrics);
+    process.nextTick(() => resultCallback({ code: ExportResultCode.SUCCESS }));
+  }
+
+  async forceFlush(): Promise<void> {}
+  async shutdown(): Promise<void> {}
+
+  getPreferredAggregationTemporality(): AggregationTemporality {
+    return AggregationTemporality.CUMULATIVE;
+  }
+}
```

Comment:
Non-blocking for me, but there's another (a bit more elaborate) `TestMetricExporter` in `PeriodicExportingMetricReader.test.ts` that could be combined with this one.

It's probably a bit out of scope for this PR, but I can open a follow up PR to cut down on some duplication. :slightly_smiling_face:

Reply:
Yeah, that one contains lots of custom functionalities that we don't have a common use case outside of `PeriodicExportingMetricReader.test.ts` so I didn't merge that one.

Reply:
Alright, good to know, thanks. Then we can leave this as is. :slightly_smiling_face: